scholia/app/templates/base.html

```diff
@@ -124,6 +124,23 @@
     });
     {% endif %}
 
+    if ("P18" in item.claims) {
+      console.log("in")
```

Comment:
I think you should remove the console.log

Reply:
yep sorry, this was actually why I added the ` * [ ] I have removed debug statements` to the PR template :)